Vagrantfile

```diff
@@ -3,17 +3,25 @@
 
 Vagrant.require_version ">= 1.8"
 
+MOUNT_OPTIONS = if Vagrant::Util::Platform.linux? then
+                  ['rw', 'vers=4', 'tcp', 'nolock']
+                else
+                  ['vers=3', 'udp']
+                end
 Vagrant.configure(2) do |config|
   config.vm.box = "ubuntu/trusty64"
 
   config.vm.synced_folder "~/.aws", "/home/vagrant/.aws"
   config.vm.synced_folder "~/.ivy2", "/home/vagrant/.ivy2"
+  config.vm.synced_folder "./", "/vagrant", type: "nfs", mount_options: MOUNT_OPTIONS
```

Comment:
This reminds me of our discussion earlier today where `.` was being mounted multiple times in different areas via different mechanisms. Here we are likely skirting that issue because we are overriding the default.